src/main/scala/tilelink/Broadcast.scala

```diff
@@ -17,6 +17,12 @@ case class TLBroadcastControlParams(
   address:   AddressSet,
   beatBytes: Int)
 
+/**
+  * @param lineBytes cacheline size
+  * @param numTrackers maximum transaction size to record on each port.
```

Comment:
i would say "the number of outstanding transactions per bank". "maximum transaction size" sounds to me like "maximum number of bytes per transaction"

Reply:
changed to the number of outstanding transactions per bank, aka MSHR sizes.